src/utils/torch_tensor_buffer.py

```diff
@@ -45,7 +47,7 @@ def _double_capacity():
         assert value.shape == self._entry_shape
 
         self._buffer[self._current_size, :] = value
-        self._current_size = self._current_size + 1
+        self._current_size = self.current_size + 1
```

Comment:
Supposed to be incrementing a variable - instead this accesses a variable that doesnt exist?

Reply:
typo - not sure how this happened tbh